macros/macros.in

```diff
@@ -234,14 +234,14 @@ package or when debugging this package.\
 %_exec_prefix		%{_prefix}
 %_bindir		%{_exec_prefix}/bin
 %_sbindir		%{_exec_prefix}/sbin
-%_libexecdir		%{_exec_prefix}/libexec
+%_libexecdir		%{_exec_prefix}/lib
 %_datadir		%{_prefix}/share
 %_docdir		%{_datadir}/doc
 %_sysconfdir		/etc
 %_sharedstatedir	%{_prefix}/com
 %_localstatedir		/var
 %_lib			lib
-%_libdir		%{_exec_prefix}/%{_lib}
+%_libdir		%{_exec_prefix}/%{_lib}/%({__dpkg_architecture} -qDEB_HOST_MULTIARCH)
```

Comment:
Your definition of `%_libdir` produces the error message
```
sh: 1: {__dpkg_architecture}: not found
```
for any use of this macro in my specfiles of
* [cwebbin](https://github.com/ascherer/cwebbin/blob/master/cwebbin.spec#L48)
* [sgb](https://github.com/ascherer/sgb/blob/local/sgb.spec#L58)
* [debbuild](https://github.com/ascherer/debbuild/blob/master/debbuild.spec#L56)

I'd rather _not_ extend `/usr/lib`.